### ISSUE TITLE
Spelling - Ancient Ore Armor: "magischem" (DE)

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#131]
**Describe the bug**
In the German localization there's a typo in the Ancient Ore Armor's description.

**Changelog**
In der Beschreibung der Antiken Erzrüstung die Phrase "[...] aus magischen Erz geschmiedet." korrigiert zu "[...] aus magischem Erz geschmiedet."

**Comment by AmProsius** [comment]
Reference:
https://github.com/AmProsius/gothic-1-community-patch/blob/2775fbc70322594ed1d00c96717143f0c2af5af7/scriptbase/_work/Data/Scripts/Content/Items/Armor.d#L1117

---

```d
TEXT[0]					=	"Diese uralte Rüstung wurde vollständig aus magischen Erz geschmiedet.";
```

changed to

```d
TEXT[0]					=	"Diese uralte Rüstung wurde vollständig aus magischem Erz geschmiedet.";
```

**Comment by AmProsius** [comment]
This issue will be fixed by #201 and can therefor be closed.
